scripts/build.sh

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+script_name="${BASH_SOURCE:-$0}"
+script_path=$(realpath "$script_name")
+scripts_dir_path=$(dirname "$script_path")
+
+# shellcheck disable=SC1091
+source "${scripts_dir_path}/lib/_functions.sh"
+# shellcheck disable=SC1091
+source "${scripts_dir_path}/lib/_logger.sh"
+
+function usage {
+    cat <<EOF
+This script performs the following tasks in this specific order:
+1. Generates a list of Operator manifests for a SonataFlow project
+2. Builds the workflow image using 
```

Comment:
this sentence is missing its end :-)

Reply:
I left that blank as an exercise for the reader 😉